docs/osf-trivia-list/dasharo.md

```diff
@@ -235,6 +235,81 @@ Dasharo Tools Suite
 * We provide ready to use binaries with GPG based signing scheme that improve
   verification where firmware coming from.
 
+## What are the differences between the official coreboot repository and the Dasharo repository?
+
+* Dasharo focus on specific platforms listed in  [supported
+  hardware](https://docs.dasharo.com/unified/novacustom/overview/) section of Dasharo
+  Universe documentation.
+
+Dasharo repository contains release tags which are associated with Dasharo
+Certification Program providing certain quality criteria including test
+results. We always trying to minimize delta, but sometimes it can be up to 5k
+SLOC (or more I guess e.g. Talos II coreboot support).
+
+## What is Dasharo Certification Program?
+
+Dasharo Certification Program (DCP) is a rigorous certification program that
+sets a high-quality bar for open-source firmware. It aims to provide stable,
+secure, and reliable firmware that meets the Dasharo values. The program
+encourages developers to create their forks of Dasharo or contribute back to
+the Dasharo project or coreboot upstream. By doing so, they can leverage the
+power of open-source development to create custom firmware tailored to their
+specific needs. The certification process ensures that the firmware binary
+meets the strict criteria by testing  3mdeb in the certification lab.
+
+The Dasharo-certified firmware provides long-term maintenance over ten years
+after the CPU microarchitecture release, which means that OEM, ODM, hardware
+vendors, and other companies can rely on the firmware for a long time without
+worrying about end-of-life issues. Moreover, DCP-approved firmware vendors must
```

Comment:
I do not quite get it who is the firmware vendor in this case

Reply:
Long term vision is that anyone can get Dasharo Certification if one pass certain criteria. So firmware vendor is any company that develop and maintain firmware and want to get Dasharo Certification.

Reply:
@macpijan do you expect changes here?